mpa/cls/stage.py

```diff
@@ -74,7 +74,10 @@ def configure(self, model_cfg, model_ckpt, data_cfg, training=True, **kwargs):
                 cfg.data.num_classes = len(cfg.data.train.get('classes', []))
             cfg.model.head.num_classes = cfg.data.num_classes
 
-        if isinstance(cfg.model.head.topk, tuple):
+        if 'Multi' in cfg.model.head.get('type', False) and cfg.model.head.get('topk', False):
```

Comment:
Can't we use cfg.task as the standard (not head.type)?
I think we can also combine this if statement with 79-80 line and use it as one if statement.

Reply:
sorry, I was mistaken. This is using the same cfg.task as 'classification'.
I think you can change cfg.task to multi-label classification from classification 
or The conditional statement {'Multi' in cfg.model.head.get('type') }needs to be reconsidered. (need to be more specific.)